### ISSUE TITLE
Afking in the main menu.

### ISSUE THREAD

**Issue by LIMUNADE** [joshuaptfan/KF2-drop-farm#3]
Instead of afking in the main menu you could waste this 30 minutes idling to gain vault dosh. There is a map called "KF-Idle_Map" (https://gamebanana.com/maps/199475). 

So, in your script when you launch the game you could make that game immediately load this map and idle there for 30 minutes and eventually die to get a loot.

**Comment by npang** [comment]
This is a pretty cool idea. I think I was able to get this to work by adding in the following lines right below the app launch & wait (line 107 I believe):

WshShell.SendKeys "{F3}open KF-Idle_map?Game=KFGameContent.KFGameInfo_Endless?Difficulty=3{ENTER}"
    WScript.Sleep 7000
    WshShell.SendKeys "{F3}startfire{ENTER}"

The 7000ms sleep command is just to buffer the map load and executing the ready up command. Googling around it looks like you can't skip trader without enabling cheats so you're stuck waiting a minute between rounds.

This only works when the script is set to launch KF2 and wait of course...

**Comment by LIMUNADE** [comment]
Thanks for updating it!

**Comment by npang** [comment]
I should clarify that I'm not the coder for this project - but you can add those lines in yourself if you want semi-functionality with the KF-idle_map!

**Comment by LIMUNADE** [comment]
Yeah, I noticed that. For some reason it's not working for me. Don't know why.

**Comment by npang** [comment]
The map needs to be installed in the location specified on that download page, but it should work normally on game launch.
